src/Compilers/CSharp/Test/Symbol/SymbolDisplay/ObjectDisplayTests.cs

```diff
@@ -203,16 +203,19 @@ public void CultureInvariance()
                 var decimalValue = new Decimal(12.5);
                 Assert.Equal("12,5", decimalValue.ToString());
                 Assert.Equal("12.5", ObjectDisplay.FormatLiteral(decimalValue, ObjectDisplayOptions.None));
+                Assert.Equal("12,5", ObjectDisplay.FormatLiteral(decimalValue, ObjectDisplayOptions.UseCurrentCulture));
```

Comment:
Can you please add a test that passes both `UseCurrentCulture` and `UseHexadecimal`?  It looks like hex ignores the current culture.  I'll tentatively assume that that's by design, but we should probably test/document it.